mail-client/alot/alot-0.8.1.ebuild

```diff
@@ -0,0 +1,88 @@
+# Copyright 1999-2019 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6} )
+
+inherit distutils-r1
+
+DESCRIPTION="Experimental terminal UI for net-mail/notmuch written in Python"
+HOMEPAGE="https://github.com/pazz/alot"
+# SRC_URI="https://github.com/pazz/alot/tarball/${PV} -> ${P}.tar.gz"
+SRC_URI="https://github.com/pazz/alot/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc test"
+
+DEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	doc? ( dev-python/sphinx[${PYTHON_USEDEP}] )
+	test? ( dev-python/mock[${PYTHON_USEDEP}] )
+	"
+RDEPEND="
+	>=app-crypt/gpgme-1.9.0[python,${PYTHON_USEDEP}]
+	>=dev-python/configobj-4.7.0[${PYTHON_USEDEP}]
+	>=dev-python/urwid-1.3.0[${PYTHON_USEDEP}]
+	>=dev-python/urwidtrees-1.0[${PYTHON_USEDEP}]
+	>=net-mail/notmuch-0.27[crypt,python]
+	dev-python/python-magic[${PYTHON_USEDEP}]
+	dev-python/twisted[${PYTHON_USEDEP}]
+	net-mail/mailbase
+	"
+
+python_prepare_all() {
+	find "${S}" -name '*.py' -exec sed -e '1i# -*- coding: utf-8 -*-' -i '{}' +
```

Comment:
Is this reported upstream?

Reply:
I am not sure we need that so I did not opened and issue for that,
we probably can get rid of it, so I removed it.